src/jdk.jfr/share/classes/jdk/jfr/internal/tool/Metadata.java

```diff
@@ -99,7 +106,11 @@ public String getName() {
 
     @Override
```

Comment:
I think ine would like to get similar help as for the ''print' command when it comes to filtering.

Reply:
Added.